integration-tests/src/test/java/org/apache/fineract/integrationtests/common/loans/InterestPauseHelper.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.fineract.integrationtests.common.loans;
+
+import io.restassured.specification.RequestSpecification;
+import io.restassured.specification.ResponseSpecification;
+import org.apache.fineract.integrationtests.client.IntegrationTest;
+import org.apache.fineract.integrationtests.common.Utils;
+
+public class InterestPauseHelper extends IntegrationTest {
+
+    private static final String BASE_URL = "/fineract-provider/api/v1/loans/";
+    private final RequestSpecification requestSpec;
+    private final ResponseSpecification defaultResponseSpec;
+
+    public InterestPauseHelper(final RequestSpecification requestSpec, final ResponseSpecification defaultResponseSpec) {
```

Comment:
Please use the `fineract-client` API instead!
Example:
`org.apache.fineract.integrationtests.common.loans.LoanTransactionHelper#makeLoanRepayment(java.lang.Long, org.apache.fineract.client.models.PostLoansLoanIdTransactionsRequest)`

Reply:
Fixed